From ba847599c6e147843eba7a3c09879287ee752976 Mon Sep 17 00:00:00 2001
From: Aswin M <Aswin_M@comcast.com>
Date: Wed, 1 Jun 2022 06:45:41 +0000
Subject: [PATCH] HX44X-149 : Bring in Voice Support

Reason for change: Modified SmartScreen overlay configurations for xr-speech-avs.
Test Procedure: Enable necessary configurations and build.
Risks: Low
License: Inherited
Upstream-Status: Pending
Source: RDKM

Signed-off-by: Aswin M <Aswin_M@comcast.com>
---
 SmartScreenControl.json | 20 +++++++++++---------
 1 file changed, 11 insertions(+), 9 deletions(-)

diff --git a/SmartScreenControl.json b/SmartScreenControl.json
index 83b2ab8..e8a9189 100644
--- a/SmartScreenControl.json
+++ b/SmartScreenControl.json
@@ -1,19 +1,21 @@
 {
   "start-overlay": [
-  "{\"jsonrpc\":\"2.0\", \"id\":3, \"method\":\"org.rdk.RDKShell.1.launch\", \"params\": {\"callsign\":\"SmartScreenApp\", \"type\":\"application\",\"visible\":false, \"uri\":\"http://127.0.0.1:50050/alexa-smart-screen/\"}}",
-  "{\"jsonrpc\":\"2.0\", \"id\":3, \"method\":\"org.rdk.RDKShell.1.moveToFront\", \"params\":{ \"client\": \"SmartScreenApp\" }}"
+  "{\"jsonrpc\":\"2.0\", \"id\":3, \"method\":\"org.rdk.RDKShell.1.launch\", \"params\": {\"callsign\":\"HtmlApp\", \"type\":\"application\",\"visible\":false, \"uri\":\"http://127.0.0.1:50050/alexa-smart-screen/\"}}",
+  "{\"jsonrpc\":\"2.0\", \"id\":3, \"method\":\"org.rdk.RDKShell.1.moveToFront\", \"params\":{ \"client\": \"HtmlApp\" }}",
+  "{\"jsonrpc\":\"2.0\", \"id\":3, \"method\":\"org.rdk.RDKShell.1.setFocus\", \"params\":{ \"client\": \"ResidentApp\" }}"
 ],
   "show-overlay": [
-  "{\"jsonrpc\":\"2.0\", \"id\":3, \"method\":\"org.rdk.RDKShell.1.moveToFront\", \"params\":{ \"client\": \"SmartScreenApp\" }}",
-  "{\"jsonrpc\":\"2.0\", \"id\":3, \"method\":\"org.rdk.RDKShell.1.setOpacity\", \"params\":{ \"client\": \"SmartScreenApp\", \"opacity\": 97}}",
-  "{\"jsonrpc\":\"2.0\", \"id\":3, \"method\":\"org.rdk.RDKShell.1.setVisibility\", \"params\":{ \"client\": \"SmartScreenApp\", \"visible\": true}}"
+  "{\"jsonrpc\":\"2.0\", \"id\":3, \"method\":\"org.rdk.RDKShell.1.moveToFront\", \"params\":{ \"client\": \"HtmlApp\" }}",
+  "{\"jsonrpc\":\"2.0\", \"id\":3, \"method\":\"org.rdk.RDKShell.1.setOpacity\", \"params\":{ \"client\": \"HtmlApp\", \"opacity\": 97}}",
+  "{\"jsonrpc\":\"2.0\", \"id\":3, \"method\":\"org.rdk.RDKShell.1.setVisibility\", \"params\":{ \"client\": \"HtmlApp\", \"visible\": true}}"
 ],
   "dimm-overlay": [
-  "{\"jsonrpc\":\"2.0\", \"id\":3, \"method\":\"org.rdk.RDKShell.1.moveToFront\", \"params\":{ \"client\": \"SmartScreenApp\" }}",
-  "{\"jsonrpc\":\"2.0\", \"id\":3, \"method\":\"org.rdk.RDKShell.1.setOpacity\", \"params\":{ \"client\": \"SmartScreenApp\", \"opacity\": 80}}",
-  "{\"jsonrpc\":\"2.0\", \"id\":3, \"method\":\"org.rdk.RDKShell.1.setVisibility\", \"params\":{ \"client\": \"SmartScreenApp\", \"visible\": true}}"
+  "{\"jsonrpc\":\"2.0\", \"id\":3, \"method\":\"org.rdk.RDKShell.1.moveToFront\", \"params\":{ \"client\": \"HtmlApp\" }}",
+  "{\"jsonrpc\":\"2.0\", \"id\":3, \"method\":\"org.rdk.RDKShell.1.setOpacity\", \"params\":{ \"client\": \"HtmlApp\", \"opacity\": 80}}",
+  "{\"jsonrpc\":\"2.0\", \"id\":3, \"method\":\"org.rdk.RDKShell.1.setVisibility\", \"params\":{ \"client\": \"HtmlApp\", \"visible\": true}}"
 ],
  "hide-overlay": [
- "{\"jsonrpc\":\"2.0\", \"id\":3, \"method\":\"org.rdk.RDKShell.1.setVisibility\", \"params\":{ \"client\": \"SmartScreenApp\", \"visible\": false}}"
+ "{\"jsonrpc\":\"2.0\", \"id\":3, \"method\":\"org.rdk.RDKShell.1.setVisibility\", \"params\":{ \"client\": \"HtmlApp\", \"visible\": false}}",
+ "{\"jsonrpc\":\"2.0\", \"id\":3, \"method\":\"org.rdk.RDKShell.1.moveToBack\", \"params\":{ \"client\": \"HtmlApp\" }}"
 ]
 }
-- 
2.17.1

